src/hooks/useTopTokens.ts

```diff
@@ -283,7 +283,7 @@ const FAKE_TOP_TOKENS_RESULT = {
 }
 
 const useTopTokens = (): UseTopTokensResult => {
-  const [data, setData] = useState<TokenData | null>(null)
+  const [data, setData] = useState<TokenData>({})
```

Comment:
this seems non-ideal, why this change?

Reply:
oh i thought you told me to do this LOL

Reply:
but i understand now ok i will change it back